### ISSUE TITLE
cvmfs-atlas cannot mount mount /cvmfs/atlas.cern.ch, /cvmfs/atlas-condb.cern.ch

### ISSUE THREAD

**Issue by cshimmin** [hepsw/docks#7]
When running cvmfs-atlas, the following errors occur at startup:
```
CernVM-FS: loading Fuse module... done
CernVM-FS: mounted cvmfs on /cvmfs/cernvm-prod.cern.ch
CernVM-FS: running with credentials 499:497
CernVM-FS: loading Fuse module... done
CernVM-FS: mounted cvmfs on /cvmfs/sft.cern.ch
Failed to grab mountpoint (2)
Failed to grab mountpoint (2)
::: mounting FUSE... [done]
```

I started a container with `--entrypoint bash` and found that even without any of the startup scripts running (via cubie), the `/cvmfs/atlas*` directories are missing. I've concluded that the reason is because of the `VOLUME` export in cvmfs-base; I'll send a PR.

**Comment by cshimmin** [comment]
Should be fixed by #8.

**Comment by sbinet** [comment]
thanks. fixed w/ 5fa2569
